src/org/labkey/test/tests/wiki/WikiTest.java

```diff
@@ -198,6 +198,33 @@ public void testShowPageTreeForWiki()
         assertElementPresent(Locator.id("wiki-toc-tree").append(Locator.linkContainingText(WIKI_PAGE_TITLE + " (" + WIKI_PAGE_NAME + ")")));
     }
 
+    /*
+        Regression coverage for
+        https://www.labkey.org/home/Developer/issues/Secure/issues-details.view?issueId=48019
+
+     */
+    @Test
+    public void testWikiWithComma()
+    {
+        String wikiName = "Wiki with comma's";
+        String wikiTitle = "Comma in the content";
+        String wikiContent = "This is my HTML, with commas";
+
+        goToProjectHome();
+        log("Creating the wiki " + wikiTitle);
+        WikiHelper wikiHelper = new WikiHelper(this);
+        wikiHelper.createNewWikiPage("HTML");
+        numberOfWikiCreated++;
+        setFormElement(Locator.name("name"), wikiName);
+        setFormElement(Locator.name("title"), wikiTitle);
+        wikiHelper.setWikiBody("<p>" + wikiContent + "</p>");
+        wikiHelper.saveWikiPage();
+
+
+        searchFor(PROJECT_NAME, "commas", numberOfWikiCreated, wikiTitle);
```

Comment:
This search should only show one result, right?
```suggestion
        searchFor(PROJECT_NAME, "commas", 1, wikiTitle);
```